### DISCUSSION TITLE
Option to select all assets in folder view

### DISCUSSION THREAD

**Discussion by EMP83** [immich-app/immich#12263] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Let's say that you are a photo editor, that you use Lightroom and you have already created a well-defined folder structure with many folders and many photos in the same folder.
It would be great if in the new "Folders" function we had the option to "select all"  assets from a folder and to and add them to an album or the same selection system from the timeline. Everything would be much simpler for example when you have a folder with over 1000 images and you want them all in an album than to select them from timeline.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by petersuba** [comment]
I think ideally what we'd need is the ability to multi-select just like we have in the timeline view, then the options to manipulate the selected assets just like in the timeline view.

In my example I have similarly historical time based folder structure, but many of the (older) assets have incorrect metadata and so in the timeline view they are all over the place. Going into the folder structure, multi-select (or select all) and then apply a common correct date, tag location etc. would be the ideal action but at the moment you'd need to do this one by one which is a non-starter for large libraries.

**Comment by bo0tzz** [comment]
#9159

**Reply by nielstron** [reply]
reopened in #14307
